tower-fallback/Cargo.toml

```diff
@@ -2,7 +2,9 @@
 name = "tower-fallback"
 version = "0.2.40"
 authors = ["Zcash Foundation <zebra@zfnd.org>"]
-license = "MIT"
+description = "A Tower service combinator that sends requests to a first service, then retries processing on a second fallback service if the first service errors."
+license = "MIT OR Apache-2.0"
```

Comment:
`tower-fallback` probably wasn't copied from `linkerd` or `tower`, I can't find any similar code in those repositories.

So this license change should be ok, because the Foundation owns all this code and its copyrights.